django/db/backends/postgresql/schema.py

```diff
@@ -192,10 +192,10 @@ def _delete_index_sql(self, model, name, sql=None, concurrently=False):
     def _create_index_sql(
         self, model, fields, *, name=None, suffix='', using='',
         db_tablespace=None, col_suffixes=(), sql=None, opclasses=(),
-        condition=None, concurrently=False,
+        condition=None, concurrently=False
```

Comment:
Trailing commas are there to minimize future diffs.

Reply:
Thanks, missed this after removing the `expressions` argument after refactoring.